be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/validators/MessageValidator.scala

```diff
@@ -33,4 +40,46 @@ object MessageValidator extends ContentValidator {
       Left(rpcMessage)
     }
   }
+
+  /**
+   * checks whether the sender of the JsonRpcRequest is in the attendee list inside the LAO's data
+   * @param sender    the sender we want to verify
+   * @param channel   the channel we want the LaoData for
+   * @param dbActor   the AskableActorRef we use (by default the main DbActor, obtained through getInstance)
+   */
+  def validateAttendee(sender: PublicKey, channel: Channel, dbActor: AskableActorRef = DbActor.getInstance): Boolean = {
```

Comment:
I wanted to have working code that passes the CI first in order to have to change less stuff if something went wrong. When this branch is merged, adding the checks will be a matter of 10 minutes and can be the subject of a separate PR so that they don't become too long